data.json

```diff
@@ -532,5 +532,12 @@
     "githubUsername": "adityaoberai",
     "imageUrl": "https://user-images.githubusercontent.com/31401437/106563296-974b8f80-6551-11eb-8753-a13ea9e944c2.png",
     "issueNumber": 304
+  },
+  {
+    "name": "Pascal Ojinnaka",
+    "githubUsername": "passy4ucj",
+    "imageUrl": "https://user-images.githubusercontent.com/56399504/107071622-46ec6000-67e5-11eb-8ce8-dc2b06f16dae.PNG",
+    "issueNumber": 305
```

Comment:
#305 is a pull request. Did you mean to add #336?

Reply:
Yes. I think it's a typo. Thank you